packit_service/worker/build/copr_build.py

```diff
@@ -620,6 +620,34 @@ def create_copr_project_if_not_exists(self) -> str:
             for field, (old, new) in ex.fields_to_change.items():
                 table += f"| {field} | {old} | {new} |\n"
 
+            chroots_diff = ""
+            if "chroots" in ex.fields_to_change:
+                old_chroots, new_chroots = ex.fields_to_change["chroots"]
+                chroots_diff = "Diff of chroots:\n```diff\n"
+                extra_chroots = set(old_chroots).difference(new_chroots)
+                missing_chroots = set(new_chroots).difference(old_chroots)
+                for e in extra_chroots:
+                    chroots_diff += f"-{e}\n"
+                for m in missing_chroots:
+                    chroots_diff += f"+{m}\n"
+                chroots_diff += "```\n"
+
+            if owner == "packit":
+                # the problem is on our side and user cannot fix it
+                msg = (
+                    f"Settings of a Copr project {owner}/{self.job_project} need to be updated, "
+                    "but there was an issue while Packit performed the change.\n"
```

Comment:
Hmm, it looks to me like **we** have some unfixed issue. (Or is it unclear for a reason?)

```suggestion
                    f"Settings of a Copr project {owner}/{self.job_project} need to be updated, "
                    "but Packit can't do that when there are previous builds still in progress..\n"
```

Reply:
Sorry for being a nitpicker, but can you please make a constant from this as we do for other comments?

Reply:
well, there are many strings here that are not constants but I agree to do that for my case to not increase tech debt